### PR TITLE
Update Clojure, Dart, Lisp, Scheme to alpha

### PR DESCRIPTION
**Pull request by adamkvitek**

### CODE REVIEW

**Comment by adamkvitek** [comment]
@mjambon @aryx please, can you check this PR? :)

**Comment by adamkvitek** [comment]
I cannot assign reviewers here.

File: lang.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "name": "Dart",
     "keys": ["dart"],
     "exts": [".dart"],
-    "maturity": "develop",
+    "maturity": "alpha",
```
Comment (aryx) [inline lang.json:57]: not this one. still develop.

**Review by aryx — APPROVED** [review]

File: lang.json (hunk 1 above)
Comment (adamkvitek) [reply lang.json:57]:
```suggestion
    "maturity": "develop",
```

**Review by aryx — APPROVED** [review]

**Pull request merged**